bundles/org.openhab.binding.worxlandroid/README.md

```diff
@@ -0,0 +1,649 @@
+
+*2: Values for **status** Channel:
+
+UNKNOWN, IDLE, HOME, START_SEQUENCE, LEAVING_HOME, FOLLOW_WIRE, SEARCHING_HOME, SEARCHING_WIRE, MOWING, LIFTED, TRAPPED, BLADE_BLOCKED, DEBUG, REMOTE_CONTROL,  GOING_HOME, ZONE_TRAINING, BORDER_CUT, SEARCHING_ZONE, PAUSE, MANUEL_STOP
+
+##### config
+
+| Channel   | Type     | ChannelName      |
+|-----------|----------|------------------|
+| timestamp | DateTime | config#timestamp |
+| command   | Number   | config#command   |
+
+##### multi-zones
+
+If Multi Zones are supported, you are able to define 4 separate zones and split working times by 10 to those.
+
+To ease zone configuration, you are able to set distance in meters where a specific zone starts. Bearing in mind that you roughly shall know how many meters of cable have been used (without buffer).
+
+As second step you are able to set time in percent and split in parts of 10 between allocation zones.
+
+| Channel      | Type          | ChannelName              |
+|--------------|---------------|--------------------------|
+| enable       | Switch        | multi-zones#enable       |
+| last-zone    | Number        | multi-zones#last-zone    |
+| zone-1       | Number:Length | multi-zones#zone-1       |
+| zone-2       | Number:Length | multi-zones#zone-2       |
+| zone-3       | Number:Length | multi-zones#zone-3       |
+| zone-4       | Number:Length | multi-zones#zone-4       |
+| allocation-0 | Number        | multi-zones#allocation-0 |
+| allocation-1 | Number        | multi-zones#allocation-1 |
+| allocation-2 | Number        | multi-zones#allocation-2 |
+| allocation-3 | Number        | multi-zones#allocation-3 |
+| allocation-4 | Number        | multi-zones#allocation-4 |
+| allocation-5 | Number        | multi-zones#allocation-5 |
+| allocation-6 | Number        | multi-zones#allocation-6 |
+| allocation-7 | Number        | multi-zones#allocation-7 |
+| allocation-8 | Number        | multi-zones#allocation-8 |
+| allocation-9 | Number        | multi-zones#allocation-9 |
+
+
+##### schedule
+
+| Channel        | Type     | ChannelName             |                   |
+|----------------|----------|-------------------------|-------------------|
+| mode           | String   | schedule#mode           | ONLY IF SUPPORTED |
+| time-extension | Number   | schedule#time-extension |                   |
+| next-start     | DateTime | schedule#next-start     |                   |
+| next-stop      | DateTime | schedule#next-stop      |                   |
+
+
+##### aws
+
+| Channel   | Type   | ChannelName   |
+|-----------|--------|---------------|
+| poll      | Switch | aws#poll      |
+| connected | Switch | aws#connected |
+
+
+##### sunday (Slot 1)
```

Comment:
This is quite extensive. If you add some lines that each day of the week has a slot 1 and 2 and provide a sample for just one day, it would be enough good enough to understand.

Reply:
Changed